package in.co.technoflask.projects.desiwanderer.kafka.annotation;

import in.co.technoflask.projects.desiwanderer.kafka.config.KafkaConsumerFactoryBuilderConfig;
import java.lang.annotation.Documented;
import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;
import org.springframework.context.annotation.Import;

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Import(KafkaConsumerFactoryBuilderConfig.class)
public @interface EnableKafkaConsumerFactoryBuilder {}
